pkg/stis/calstis/lib/mkpkg

```diff
@@ -11,67 +11,67 @@ install:
 	;
 
 calstis.a:
-	add2d.c		<c_iraf.h> <hstio.h> ../stis.h ../stiserr.h ../stisdef.h
+	add2d.c		<c_iraf.h> <hstio.h> ../stis.h ../../../../include/err.h ../stisdef.h
```

Comment:
@jhunkeler I think the other relative paths are incorrect(?) Is this file even still needed or can we just delete it?